webpack/webpack-embedded-dependencies-plugin/src/EmbeddedDependenciesWebpackPlugin.ts

```diff
@@ -0,0 +1,361 @@
+import path from 'path';
+import { Async, IPackageJson, Sort } from '@rushstack/node-core-library';
+import { LegacyAdapters, FileSystem } from '@rushstack/node-core-library';
+import { Compilation, Compiler, WebpackPluginInstance, sources, WebpackError } from 'webpack';
```

Comment:
```suggestion
import type { Compilation, Compiler, WebpackPluginInstance, sources, WebpackError } from 'webpack';
```

Reply:
Compilation and WebpackError and sources have to be values but I did separate type-only imports from value imports.